specification/columns/capacityreservationstatus.md

```diff
@@ -2,6 +2,20 @@
 
 Capacity Reservation Status indicates whether the charge represents either the consumption of the [*capacity reservation*](#glossary:capacity-reservation) identified in the CapacityReservationId column or when the *capacity reservation* is unused.
 
+---
+The CapacityReservationStatus column adheres to the following requirements:
+
+* CapacityReservationStatus MUST be present in a [*FOCUS dataset*](#glossary:FOCUS-dataset) when the provider supports *capacity reservations*.
+* If present, the column MUST conform to the following additional requirements:
```

Comment:
```suggestion
* If present, the column adheres to the following additional requirements:
```

Reply:
suggested change (MUST conform -> adheres) applied across the board